### ISSUE TITLE
New Mockito release removed deprecated classes.

### ISSUE THREAD

**Issue by Ferada** [szpak/mockito-java8#8]
Just noticed due to a failing build: The newest Beta release of Mockito (2.0.54-beta) removed some deprecated classes, c.f. [this commit](https://github.com/mockito/mockito/commit/dff5510cb3e97dae414e58ba85825ad01b273c72) and breaks this package with at least `DeprecatedOngoingStubbing` (that's the one I've noticed in the logs).

**Comment by nickheniser** [comment]
If you are using gradle you can workaround by forcing mockito core to specific release:

```
testCompile ('org.mockito:mockito-core:2.0.44-beta')
{
	force = true
}
```

**Comment by szpak** [comment]
Thanks for the report.

@nickheniser Yup. Alternatively `mockito-core` transitive dependency can be excluded (to not have the `forced` version):
```
testCompile('info.solidsoft.mockito:mockito-java8:2.0.0-beta.2') {
    exclude module: 'mockito-core' 
}
```

**Comment by szpak** [comment]
Btw, I have set up daily build on Jenkins, to get automatically notified about further incompatible changes (at least about some of them).

**Comment by Ferada** [comment]
Yeah I just did kinda the same thing with Maven too for the time being.

**Comment by szpak** [comment]
Implemented in 2.0.0-beta.3.
